### ISSUE TITLE
[mdc-textfield] Input value jumps up to floating label when using mdc-textfield--fullwidth.

### ISSUE THREAD

**Issue by YUzhva** [material-components/material-components-web#506]
### MDC-Web Version:
0.8.0

### What browser(s) is this bug affecting?
Chrome v57.

### What are the steps to reproduce the bug?
http://material-components-web.appspot.com/textfield.html

Inspect first email address text field with chrome dev. tools.
Add `mdc-textfield--fullwidth` class to the `mdc-textfield` div tag
```html
<div class="mdc-textfield mdc-textfield--upgraded">
...
```

So it will be like this:
```html
<div class="mdc-textfield mdc-textfield--upgraded mdc-textfield--fullwidth">
    <input
        type="text"
        class="mdc-textfield__input"
        id="my-textfield"
        name="email"
        aria-controls="my-textfield-helptext"
        data-demo-no-auto-js=""
        autocomplete="email">

    <label
        for="my-textfield"
        class="mdc-textfield__label mdc-textfield__label--float-above">
        
        Email Address
    </label>
</div>
```

P.S: Also, you could try to remove class `mdc-textfield--fullwidth` from Pen: http://codepen.io/anon/pen/jmNYbg

### What is the expected behavior?
On typing text, input text field value should not jump  up to the label.
Also, looks like the bottom line doesn't change its color when I'm typing text.

### What is the actual behavior?
Input value jumps up to the blue label.
Bottom line should change color when input is active.

### Screenshot
![alt tag](https://lh5.googleusercontent.com/wEAvxiw-wE2ddnjRMwaEHeDwmYlbCUH9BzH6PhWtwRK5gLzYDnWe1VotlsDqcbg5tZhIeT75qfz2XKU=w2880-h1652-rw)

**Comment by berets76** [comment]
As you can see on this previous issue #203 , floating labels are not present on full-width textfields.

**Comment by yeelan0319** [comment]
As @berets76 mentioned, `mdc-textfield--fullwidth` do not have floating labels.
